docs/intro/tutorial01.txt

```diff
@@ -430,7 +430,7 @@ look like this:
         'django.contrib.messages',
         'django.contrib.staticfiles',
         'polls',
```

Comment:
Since a stated goal of moving from tuples to lists is avoiding the trailing comma problem in tuples, maybe remove the trailing comma from the list in the example?